src/main/java/taskat5/Student.java

```diff
@@ -0,0 +1,82 @@
+package taskat5;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.concurrent.TimeUnit;
+
+public class Student {
+    private final String studentName;
+    private final String specialty;
+    private final String startDate;
+    private final Course[] courses = new Course[10];
+    private int courseInd;
+    private int totalCoursesDuration = 0;
+
+    public Student(String studentName, String specialty, String startDate) {
```

Comment:
why date is a String type?

Reply:
for self-educational reason I wanted to use parse.